### DISCUSSION TITLE
Training a custom relation extraction component

### DISCUSSION THREAD

**Discussion by CarlaMPR** [explosion/spaCy#10928] [Help: Coding & Implementations] [answered]
Hi! :) I'm working on Relation Extraction, specifically the extraction of drug-drug interactions from text documents.
I saw this tutorial, https://www.youtube.com/watch?v=8HL-Ap5_Axo, and I'm trying to implement that in my specific case. I'm using this corpus, https://github.com/isegura/DDICorpus, consisting of a collection of XML files with the drugs and the relations between them already labelled. With this, I had two doubts and that's why I decided to send you this message.
1) Do I need a NER model to identify the drugs (my entities) in the first place?
2) In what format should I have the corpus? I saw this example, https://github.com/explosion/projects/blob/v3/tutorials/rel_component/assets/annotations.jsonl, but I couldn't understand what "token_start" and "token_end" means.

Thank You!

**Comment by polm** [comment, marked as answer]
Closing as duplicate of #10930. Please don't post duplicate threads.
